docs/src/main/asciidoc/picocli.adoc

```diff
@@ -88,11 +91,71 @@ public class EntryCommand {
     public void setName(String name) {
         this.name = name;
     }
-
 }
 ----
 
-== Command line application with multiple Commands
+=== Packaging
+
+Picocli provides different ways
+
+> to package your application in such a way that end users can invoke it by its command name. https://picocli.info/#_packaging_your_application[Source]
+
+If you want to distribute a single executable file that runs without installing a JDK see <<Native mode support>>. Notice that this executable will be platform specific.
```

Comment:
Please use an anchor for `<<Native mode support>>` or things will get easily broken if the title is changed.